docs/gds-supported-platform/getting-started-gsp-local.md

```diff
@@ -1,54 +1,113 @@
-# Getting started with a local GDS Supported Platform development cluster
+# Set up a local GDS Supported Platform instance
 
-These instructions tell you how to set up a local GDS Supported Platform (GSP) cluster and host an app on that cluster.
+The GDS Supported Platform (GSP) is a platform to host Government Digital Service (GDS) services. The GSP is based on [Docker](https://docs.docker.com/) and [Kubernetes](https://kubernetes.io/).
 
-This process is resource-intensive and you must set the resource amount used by Docker to the following:
-- 4 CPUs
-- 8 gigabytes of memory
+These instructions explain how to set up a local instance of the GSP and run the example `govuk-prototype-kit` app on that instance. You can use this process to test your app before deploying that app to a production environment on the GSP. This process has several stages. 
 
-## Build a local GSP cluster
+1. Install [prerequisite software](#install-prerequisite-software).
+1. [Create and run](#create-and-run-a-local-gsp-instance) a local GSP instance.
+1. Create a [Docker image of the `govuk-prototype-kit` app](#create-a-docker-image-of-the-govuk-prototype-kit-app).
+1. Create a [Helm chart](#create-a-helm-chart).
+1. [Deploy the app](#deploy-the-app-on-to-the-local-GSP-instance) on to the local GSP instance.
+1. [Connect](#connect-to-the-app) to the app.
+1. [Destroy](#destroy-the-local-instance) the local instance.
 
-1. Install [Homebrew](https://brew.sh/)
+Setting up a local GSP instance should take no more than 2 hours. Contact the GSP team using the [#re-gsp Slack channel](https://gds.slack.com/messages/CDA7YSP0D) if this process takes longer.
 
-1. Clone the GSP repo:
+<%= warning_text('While setting up or running a local GSP instance, do not:<br>- power down your laptop<br>- put your laptop in standby mode<br>- connect or disconnect the Cisco VPN client') %>
+
+
+## Install prerequisite software
+
+1. Install [Homebrew](https://brew.sh/) and run `brew update` to make sure you have the latest version of Homebrew.
+
+
+
+1. Go to the local repository folder and run `brew bundle` to install the [Homebrew Bundler](https://github.com/Homebrew/homebrew-bundle).
+
+1. Grant `driver superuser` privileges to the hypervisor:
 
     ```
-    git clone https://github.com/alphagov/gsp.git
+    sudo chown root:wheel /usr/local/bin/docker-machine-driver-hyperkit && sudo chmod u+s /usr/local/bin/docker-machine-driver-hyperkit
```

Comment:
This step is only relevant if running on MacOS as hyperkit is a MacOS thing. Are there other docs for Linux?

Reply:
Unfortunately not at the moment, the assumption is that most people are using a mac and hyperkit

Reply:
Agreed with @issyl0 to iterate once docs are being used and we have a better idea of how much of a user need this is.